src/UI/Component/Icon/Factory.php

```diff
@@ -9,13 +9,29 @@
 	 * ---
 	 * description:
 	 *   purpose: >
-	 *    Standard Icons represent ILIAS Objects.
+	 *    Standard Icons represent ILIAS Objects. In their outlined version, they are used to be layered upon a picture or
+	 *    if the Standard Icon should not too much of the users attention.
```

Comment:
I think a `catch`or something similar is missing here:

```suggestion
	 *    if the Standard Icon should not catch too much of the users attention.
```